src/Components/Users/UserProfile.tsx

```diff
@@ -182,14 +181,39 @@ export default function UserProfile() {
                 Notification.Success({
                     msg: "Details updated successfully"
                 });
-                window.location.reload();
+                setDetails({ ...details, first_name: states.form.firstName, last_name: states.form.lastName, age: states.form.age, gender: genderTypes.filter(el => { return el.id === Number(states.form.gender) })[0].text, email: states.form.email, phone_number: states.form.phoneNumber });
```

Comment:
Codacy found an issue: [Identifier 'first_name' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5838477)